app/main.py

```diff
@@ -1,2 +1,61 @@
+from dataclasses import dataclass
+from typing import Hashable
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: any
```

Comment:
The type hint for `value` is `any`, which is not specific. Consider using a more specific type hint if possible, or use `Any` from `typing` for consistency.